{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40783713",
  "multipleFaultNumbers": null,
  "date": "2020-10-28T14:30:03",
  "region": "Bolton",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-10-28T14:30:03",
  "information": "",
  "actualTimeOfRestoration": "2020-10-28T14:30:03",
  "outageCentrePoint": {
    "lat": 53.549522,
    "lng": -2.422062
  },
  "outageLocations": null,
  "AffectedPostcodes": " BL4 0HB, BL4 0JP, BL4 0JQ, BL4 0JU",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}